sdk/storage/Azure.Storage.DataMovement.Blobs/stress/src/src/DataMovementBlobStressConstants.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage.DataMovement.Blobs.Stress
+{
+    internal class DataMovementBlobStressConstants
+    {
+        public const int KB = 1024;
+        public const int MB = KB * 1024;
+        public const int GB = MB * 1024;
+        public const long TB = GB * 1024L;
+        public const int Base16 = 16;
```

Comment:
Do we not have access to our Common in stress testing? I'm sure we have these constants somewhere.

Reply:
Yeah. Actually it's not in our tests, I have to pull them from one of our storage packages